content/en/apps/tutorials/local-setup.md

```diff
@@ -37,18 +37,41 @@ To read more about these concepts, see our [Docker Setup guide]({{< relref "core
 
 Before you begin, you need to have some useful software and tools that are required for things to work.
 
-Install [nodejs](https://nodejs.org/en/) 8 or later and [npm](https://www.npmjs.com/get-npm). As well, [install `docker` and `docker-compose`]({{< relref "apps/guides/hosting/requirements#docker" >}}).
+Install [nodejs](https://nodejs.org/en/) 8 or later, [npm](https://www.npmjs.com/get-npm), [git](https://git-scm.com/downloads) or the [Github Desktop](https://desktop.github.com/) and [`docker` and `docker-compose`]({{< relref "apps/guides/hosting/requirements#docker" >}}).
 
 ## Implementation Steps
 
 Now that you have the dependent tools and software installed, you are ready to set up your CHT local environment.
  
 
-### 1. Install medic-conf
+
+### 1. Install the Core Framework
+
+Check out the [cht-core respository](https://github.com/medic/cht-core) to your local machine, either by using the [Github Desktop app](https://desktop.github.com/) or by running the following command in the directory you want to check the code out into: `git clone https://github.com/medic/cht-core.git`. This will create a `cht-core` directory.  
+
+Open your terminal or command line and navigate to `cht-core` directory which is where the `docker-compose.yml` file is. Run the command:
+
+```shell
+docker-compose up
+```
+
+{{< figure src="medic-login.png" link="medic-login.png" class="right col-6 col-lg-8" >}}
+
+Once the command is done running, navigate to [https://localhost](https://localhost) on a Google Chrome browser and login with the default username `medic` and default password `password`. You will get an error "Your connection is not private" (see [screenshot](./privacy.error.png)). Click "Advanced" and then click "Proceed to localhost". This error can be fixed in step 5 below.
```

Comment:
I would think you go to a page `in` or `with` a browser... but that might just be me!

```suggestion
Once the command is done running, navigate to [https://localhost](https://localhost) with the Google Chrome browser and login with the default username `medic` and default password `password`. You will get an error "Your connection is not private" (see [screenshot](./privacy.error.png)). Click "Advanced" and then click "Proceed to localhost". This error can be fixed in step 5 below.
```